packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -2,6 +2,7 @@ import os from 'os'
 import pc from 'picocolors'
 import { randomBytes } from '@stablelib/random'
 import * as u8a from 'uint8arrays'
+import { fromBooleanInput } from '@ceramicnetwork/ipfs-daemon'
```

Comment:
I couldn't move it since it had to the part in the same folder. My guess because of it being a package - so unless there are strong feelings around it I'll just keep as is.